roboprop_client/export_model.py

```diff
@@ -43,72 +55,76 @@ def write_xml(xml: ElementTree.Element, filepath: str):
         print(f"Saved: {filepath}")
 
     def generate_sdf():
-        sdf = ElementTree.Element("sdf", attrib={"version": sdf_version})
-        model = ElementTree.SubElement(sdf, "model", attrib={"name": model_name})
-        statit_xml = ElementTree.SubElement(model, "static")
-        statit_xml.text = str(True)
-        link = ElementTree.SubElement(
-            model, "link", attrib={"name": f"{model_name}_link"}
-        )
+        for path_sdf, path_visual in zip(sdf_paths, visual_paths):
+            sdf = ElementTree.Element("sdf", attrib={"version": sdf_version})
+            model = ElementTree.SubElement(sdf, "model", attrib={"name": model_name})
+            statit_xml = ElementTree.SubElement(model, "static")
```

Comment:
as you pointed, there is a typo, it should be `static_xml`

Reply:
ah yes, sorry and thank you for the reminder. Changed